docs/schema/examples/etcher.json

```diff
@@ -0,0 +1,133 @@
+{
+  "name": "etcher",
+  "lead": "Burn. Better.",
```

Comment:
Where is this taken from? I guess `description` comes from the GitHub repo description, but what about this one?

Reply:
Yea I'm not sure :/ we could look for a blockquote in the readme? Or just rely on input from the landr config.